### PR TITLE
Dramatic Doors 翻译提交

### PR DESCRIPTION
**Pull request by ChuijkYahus**
- [x] 我已**仔细**阅读贡献指南 [CONTRIBUTING](https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/blob/main/CONTRIBUTING.md)；
- [x] 我已确认英文原文（如 en_us.json）存在且完整，内容与中文对应；
- [x] 我已阅读并同意按 [CC BY-NC-SA 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/deed.zh) 协议发布我的作品；
- [ ] 刷新 PR 的标签/状态，有需要再点击；

### CODE REVIEW

**Comment by ChuijkYahus** [comment]
/mv-recursive "projects/1.18/assets/dramatic-doors/dramatic_doors/" "projects/1.18/assets/dramatic-doors/dramaticdoors/"

File: projects/1.18/assets/dramatic-doors/dramaticdoors/lang/zh_cn.json [hunk 1]
```diff
@@ -0,0 +1,580 @@
...
+  "block.dramaticdoors.tall_macaw_dark_oak_tropical_door": "高深色橡木热带风格门",
+  "block.dramaticdoors.tall_macaw_jungle_tropical_door": "高丛林木热带风格门",
+  "block.dramaticdoors.tall_macaw_mangrove_tropical_door": "高红树木热带风格门",
+  "block.dramaticdoors.tall_macaw_oak_tropical_door": "高橡木热带风格门",
+  "block.dramaticdoors.tall_macaw_spruce_tropical_door": "高云杉木热带风格门",
+  "block.dramaticdoors.tall_macaw_warped_tropical_door": "高诡异木热带风格门",
+  
+  "itemGroup.dramaticdoors": "Dramatic Doors",
+  "itemGroup.dramaticdoors_chipped": "Dramatic Doors丨Chipped",
+  "itemGroup.dramaticdoors_macaw": "Dramatic Doors丨Macaw的门"
```
Comment (dovisutu) [inline projects/1.18/assets/dramatic-doors/dramaticdoors/lang/zh_cn.json:579]: ~mcwdoors的译名是这个吗~

**Review by dovisutu — APPROVED** [review]
排列组合++
已经认不得门字了......

File: projects/1.18/assets/dramatic-doors/dramaticdoors/lang/zh_cn.json (hunk 1 above)
Comment (ChuijkYahus) [reply projects/1.18/assets/dramatic-doors/dramaticdoors/lang/zh_cn.json:579]: 确实（）

**Comment by ChuijkYahus** [comment]
作者真是勤劳

**Comment by ChuijkYahus** [comment]
兼容一览表（）：
兼容模组|CurseForge ID|备注
-:|:-|-
Biomes O' Plenty|biomes-o-plenty
Oh the Biomes You'll Go|oh-the-biomes-youll-go
Prehistoric Fauna|prehistoric-fauna
The Twilight Forest|the-twilight-forest
Atmospheric|atmospheric
Autumnity|autumnity
Buzzier Bees|buzzier-bees
The Outer End|the-outer-end
The Endergetic Expansion|endergetic
Environmental|environmental
Upgrade Aquatic|upgrade-aquatic
Abundance|abundance
Bayou Blues|bayou-blues
Enhanced Mushrooms|enhanced-mushrooms
Abundant Atmosphere|abundant_atmosphere
Architect's Palette|architects-palette
Ars Nouveau|ars-nouveau
Bamboo Everything|bamboo-everything
Biomancy|biomancy
Biome Makeover|biome-makeover-forge
Blocks +|shadow-x-blocks-plus
Blue Skies|blue-skies
Caupona|caupona
The Ceilands|the-ceilands
Colorful Azaleas|colorfulazaleas
Copper Overhaul|copper-overhaul
Create: Alloyed|create-alloyed
Create Deco|create-deco
Darker Depths|darker-depths|#2157
Deeper and Darker|deeperdarker
'Dustrial Decor|dustrial-decor
Ecologics|ecologics
Everything is Copper|everythingcopper
End's Phantasm|phantasm
Enlightened End|enlightened-end|#2219
Forbidden and Arcanus|forbidden-arcanus
Good Ending|good-ending
The Graveyard|the-graveyard-forge
Habitat|habitat
Hex Casting|hexcasting
Hexerei|hexerei|#2447
Integrated Dynamics|integrated-dynamics|
Malum|malum|#2319
Nether's Exoticism|nethers-exoticism
NewWorld|newworld
PokeCube AIO|pokecube-aoi
Premium Wood|premium-wood
Quark|quark
Kitteh6660's Morecraft|kitteh6660s-morecraft|#2285
Snowy Spirit|snowy-spirit
Supplementaries|supplementaries
Tinker's Construct|tinkers-construct
Twigs|twigs
The Undergarden|the-undergarden
Windswept|windsweptmod|#2424

**Review by dovisutu — APPROVED** [review]

**Pull request merged**